src/core/symbology-ng/qgsfillsymbollayerv2.h

```diff
@@ -550,7 +550,7 @@ class CORE_EXPORT QgsImageFillSymbolLayer: public QgsFillSymbolLayerV2
     virtual ~QgsImageFillSymbolLayer();
     void renderPolygon( const QPolygonF& points, QList<QPolygonF>* rings, QgsSymbolV2RenderContext& context ) override;
 
-    virtual QgsSymbolV2* subSymbol() override { return mOutline; }
+    virtual QgsSymbolV2* subSymbol() const override { return mOutline; }
```

Comment:
That's always a question with member pointers.
But since we have the ownership of it, probably yes.